h2o-docs/src/product/data-science/gbm.rst

```diff
@@ -233,6 +233,10 @@ Defining a GBM Model
    than 2. For more information, refer to `Tweedie
    distribution <https://en.wikipedia.org/wiki/Tweedie_distribution>`__.
 
+- `upload_custom_distribution <algo-params/upload_custom_distribution.html>`__: Upload a custom distribution into a running H2O cluster.
```

Comment:
There is another instance of upload_custom_distribution at the end of this list of parameters. Can you remove one of these?